From d111f18bf33c0bbfff36959014292e7a3d840547 Mon Sep 17 00:00:00 2001
From: Mike Mellenthin <mikem@cloudera.com>
Date: Fri, 28 Feb 2014 16:20:17 -0800
Subject: [PATCH 179/596] CLOUDERA-BUILD. Use a more calm color in the Hadoop
 web UI.

(cherry picked from commit 275e44425b7c7c39265837f3da5cd1af3c042b20)
(cherry picked from commit 8ba202b957f82df0623a86aca99c9f5d1c95625a)
---
 .../hadoop-hdfs/src/main/webapps/static/hadoop.css |   10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/static/hadoop.css b/hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/static/hadoop.css
index 4b0e967..818f120 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/static/hadoop.css
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/static/hadoop.css
@@ -191,25 +191,25 @@ div.security {
 
 .panel-success > .panel-heading {
   color: #fff !important;
-  background-color: #5FA33E !important;
+  background-color: #004d6f !important;
 }
 
 header.bs-docs-nav, header.bs-docs-nav .navbar-brand {
   border-radius: 0px;
-  background-color: #5fa33e;
+  background-color: #004d6f;
   color: #fff;
 }
 
 #ui-tabs > li > a {
-  color: #dcf0d3;
+  color: #fff;
 }
 
 #ui-tabs .active a {
   color: #fff;
-  background-color: #446633;
+  background-color: #00314f;
 }
 
 #alert-panel {
   margin-top:20px;
   display: none;
-}
\ No newline at end of file
+}
-- 
1.7.9.5

